clients/drcachesim/common/options.cpp

```diff
@@ -1035,5 +1043,12 @@ droption_t<bool> op_abort_on_invariant_error(
     "total invariant error count is printed at the end; a non-zero error count does not "
     "affect the exit code of the analyzer.");
 
+droption_t<bool> op_pt2ir_allow_recoverable_errors(
+    DROPTION_SCOPE_ALL, "pt2ir_allow_recoverable_errors", false,
+    "Whether recoverable errors are tolerated during PT trace decode using pt2ir",
+    "By default, errors in decoding the kernel syscall PT trace in pt2ir are "
+    "fatal. If this is true, some recoverable errors are allowed and their "
+    "counts are reported by raw2trace at the end.");
```

Comment:
This doesn't tell the user what the consequences are of these "errors": explain that this results in gaps in the trace which could cause problems with simulators.  How about instead of "error" we say "discontinuities".

Reply:
"discontinuities" are just one case. In other cases the full syscall may have to be dropped. Clarified.